misago/project_template/project_name/settings.py

```diff
@@ -411,6 +411,7 @@
 ]
 
 
-# Specifies the amout of time we can saves the user ips in days
+# Specifies the number of days that IP addresses are stored in the database before anonymization.
+# Change this setting to None to never anonymize old IP addresses.
 
-MISAGO_IP_STORE_TIME = 50
\ No newline at end of file
+MISAGO_IP_STORE_TIME = None
```

Comment:
This setting should be left at 50. We want IP anonymization enabled by default on new forums, while leaving it of for those updating from old versions.